DRAFT.md

```diff
@@ -607,6 +665,9 @@ class polymorphic {
 
   friend constexpr void swap(polymorphic& lhs, polymorphic& rhs) noexcept;
 };
+
+template <class T, class Alloc>
+struct std::uses_allocator<polymorphic<T>, Alloc> : true_type {};
```

Comment:
Q: Should this be `polymorphic<T, Alloc>` ?

Reply:
Also corrected